Simonyi/Non-Standard-Operations/MTCS/TMA/Simonyi-LOTO-procedure.rst

```diff
@@ -70,20 +75,98 @@ Procedure Steps
 .. In the case of more complicated procedures, more sophisticated methodologies may be appropriate, such as multiple section headings or a list of linked procedures to be performed in the specified order.
 .. For highly complicated procedures, consider breaking them into separate procedure. Some options are a high-level procedure with links, separating into smaller procedures or utilizing the reST ``include`` directive <https://docutils.sourceforge.io/docs/ref/rst/directives.html#include>.
 
+.. note::
```

Comment:
Maybe change this note to the first step of the procedure. 1.  Before/right after applying LOTO on the OSS, announce "LOTO on the OSS" on  *#summit-announce* and/or *#summit-simonyi*.

Reply:
done